iac-extensions/spring-config/build.gradle.kts

```diff
@@ -15,3 +18,17 @@ dependencies {
     testImplementation(libs.sonar.analyzer.test.commons)
     testImplementation(testFixtures(project(":iac-common")))
 }
+
+tasks.generateGrammarSource {
+    maxHeapSize = "64m"
+    arguments = arguments + listOf("-visitor", "-long-messages", "-no-listener", "-package", "org.sonar.iac.springconfig.parser.properties")
+    // Due to bug in ANTLR Gradle plugin https://stackoverflow.com/a/49388412
```

Comment:
I don't get what the bug is. Is it that ANTLR source is not in a package but we want the generated file to be? Then I wouldn't call it a bug and rephrase the comment like `antrl plugin requires grammar file to be placed without package in path, but we want the generated file to follow standard Java convention`